lib/src/main/java/io/ably/lib/transport/Defaults.java

```diff
@@ -2,43 +2,32 @@
 
 import io.ably.lib.types.ClientOptions;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
 public class Defaults {
-	public static final int protocolVersion     = 1;
-	public static final String[] FALLBACK_HOSTS = new String[] {"A.ably-realtime.com", "B.ably-realtime.com", "C.ably-realtime.com", "D.ably-realtime.com", "E.ably-realtime.com"};
-	public static final String REST_HOST        = "rest.ably.io";
-	public static final String REALTIME_HOST    = "realtime.ably.io";
-	public static final int PORT                = 80;
-	public static final int TLS_PORT            = 443;
-	public static final int connectTimeout      = 15000;
-	public static final int disconnectTimeout   = 30000;
-	public static final int suspendedTimeout    = 120000;
-	public static final int cometRecvTimeout    = 90000;
-	public static final int cometSendTimeout    = 10000;
-	public static final String[] transports     = new String[]{"web_socket"};
-	public static final String transport        = "io.ably.lib.transport.WebSocketTransport$Factory";
+	public static final int PROTOCOL_VERSION        = 1;
```

Comment:
What is this for?  PROTOCOL_VERSION is `0.8` anyway as far as I was aware? http://docs.ably.io/client-lib-development-guide/features/#G4